pages/learn/glossary.md

```diff
@@ -53,6 +53,10 @@ A property appearing within a [schema](#schema) object.
 
 The [JSON Schema specification](../specification) defines behavior for a large library of keywords which can be used to describe [instances](#instance).
 
+### implementation
+
+Historically, the word "implementation" has been used to describe both specifically validators/annotators (interpreted as strictly operating as the spec describes) and also various other kinds of JSON Schema tooling.  However, due to this ambiguity, we have decided that the preferred conversational term should be ["tooling"](#tooling).
```

Comment:
```suggestion
Historically, the word "implementation" has been used to describe both specifically validators/annotators and also various other kinds of JSON Schema tooling.  However, due to this ambiguity, we have decided that the preferred conversational term should be ["tooling"](#tooling).
```

Reply:
I think that note is important for context.

Reply:
I'm not suggesting the historical note be removed. My concern is that this sounds like something is not considered an implementation if it doesn't strictly conform to the spec. It sounds like ajv wouldn't be considered an implementation, which isn't accurate, even historically.

Reply:
Okay.  The note you're suggesting to remove is intended to express that the spec only covers validation/annotation.  What would you have to express that?

Reply:
My best suggestion might be "(which is what the spec describes)". The words feel a little awkward, but I think it conveys the intended meaning.

Reply:
How now?

Reply:
Perfect :+1: